content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -278,9 +278,8 @@ Init containers have all of the fields of an app container. However, Kubernetes
 prohibits `readinessProbe` from being used because init containers cannot
 define readiness distinct from completion. This is enforced during validation.
 
-Use `activeDeadlineSeconds` on the Pod and `livenessProbe` on the container to
-prevent init containers from failing forever. The active deadline includes init
-containers.
+Use `activeDeadlineSeconds` on the Pod. The active deadline includes init
```

Comment:
How about the following.
```suggestion
Use `activeDeadlineSeconds` on the Pod to prevent init containers from failing forever. The active deadline includes init
```

Reply:
updated.